pytorch_toolkit/nncf/nncf/README.md

```diff
@@ -141,91 +146,101 @@ For symmetric:
 
-**Algorithm parameters**:
-- `algorithm` - the name of the compression algorithm; should be set to `binarization` in this case.
-- `mode` - the mode of weight binarization. Should be set either to `xnor` (default) or `dorefa`.
-- `ignored_scopes` - blacklist: layers that should be excluded from binarization (optional).
+**Algorithm Parameters**:
+
+- `algorithm` - the name of the compression algorithm. Set to `binarization` in this case
+- `mode` - the mode of weight binarization. Set either to `xnor` (default) or `dorefa`
+- `ignored_scopes` - blacklist: layers that should be excluded from binarization (optional)
 - `target_scopes` - whitelist: only the layers listed here should be compressed (optional)
 - `params` - parameters of the binarization algorithm:
    - `batch_multiplier` - allows to increase the effective batch size during training by accumulating the gradients each `batch_multiplier - 1` batches and only performing the optimization step during the next training batch. Increasing this may improve training quality, since binarized networks exhibit noisy gradients requiring larger batch sizes than could be accomodated by GPUs. Default is 1.
-   - `activations_bin_start_epoch` - starting from this training epoch, the convolution activations will become binarized. Default is 1.
-   - `weights_bin_start_epoch` - starting from this training epoch, the convolution weights will become binarized. Default is 1.
-   - `lr_poly_drop_start_epoch` - (optional) starting from this training epoch, the learning rate of the optimizer will drop to 0 according to a polynomial law.
-   - `lr_poly_drop_duration_epochs` - specifies the duration of the learning rate polynomial drop stage, in training epochs. Default is 30.
-   - `disable_wd_start_epoch` - (optional) starting from this training epoch, the weight decay parameter of the optimizer will be set to 0.
+   - `activations_bin_start_epoch` - starting from this training epoch, the convolution activations become binarized. Default is 1.
+   - `weights_bin_start_epoch` - starting from this training epoch, the convolution weights become binarized. Default is 1.
+   - `lr_poly_drop_start_epoch` - (optional) starting from this training epoch, the learning rate of the optimizer drops to 0 according to a polynomial law.
+   - `lr_poly_drop_duration_epochs` - specifies the duration of the learning rate polynomial drop stage in training epochs. Default is 30.
+   - `disable_wd_start_epoch` - (optional) starting from this training epoch, the weight decay parameter of the optimizer is set to 0.
 
 
-### Non-structured sparsity
+### Non-Structured Sparsity
+
 Sparsity algorithm zeros weights in convolutional and fully-connected layers in a non-structured way, so that zero values are randomly distributed inside the tensor. Most of the sparsity algorithms set less to zero the less important weights but the criteria of how they do it is different. The framework contains several implementations of sparsity methods.
```

Comment:
@AlexKoff88 Could you please help here?
The sentence sounds a bit unclear:
`... algorithms set less to zero the less important weights...`  -- Confused about the first occurrence of `less` here. Set less what?

Reply:
This is a typo, should be:  `Most of the sparsity algorithms set the less important weights to zero but the criteria of how they do it is different.`

Reply:
@AlexKoff88 Thanks!